config/prod.webpack.config.js

```diff
@@ -10,6 +10,9 @@ const { config: webpackConfig, plugins } = config({
 plugins.push(
   require('@redhat-cloud-services/frontend-components-config/federated-modules')({
     root: resolve(__dirname, '../'),
+    exposes: {
```

Comment:
When configuring the expose object, you also have to expose the `./RootApp` module. The default config is overriden.

Reply:
Fixed